arviz/tests/test_stats.py

```diff
@@ -113,8 +113,9 @@ def test_summary_fmt(centered_eight, fmt):
     assert summary(centered_eight, fmt=fmt) is not None
 
 
-def test_summary_stat_func(centered_eight):
-    assert summary(centered_eight, stat_funcs=[np.var]) is not None
+# Issue with xarray.apply_ufunc
```

Comment:
Yeah, that sounds reasonable.

You can make a commit when you have time

Reply:
I added `@pytest.mark.xfail` for tests that are failing due to bugs. Thanks for the tip.

Reply:
If you catch new ones can you add a short snippet of what is failing as well? I can go back and do it too if you're short on time

The reason I ask is this will help others when they try to go back and fix the bug. This way when people come to contribute they both have a test and comment explaining what to look into!

Examples are here
https://docs.pytest.org/en/latest/skipping.html#examples

Reply:
Feel free to edit. I can do this later (3 hours) but if you have time, you can do it.